### PR TITLE
Adjustable dance speed

### PR DESCRIPTION
**Pull request by dylwhich**

### CODE REVIEW

File: main/modes/mode_dance.c [hunk 1]
```diff
@@ -432,6 +457,50 @@ void danceButtonCb(buttonEvt_t* evt)
     }
 }
 
+void danceTouchCb(touch_event_t* evt)
+{
+    dancePollTouch();
```
Comment (AEFeinstein) [inline main/modes/mode_dance.c:462]: Is there any feedback during the swipe?

**Comment by AEFeinstein** [comment]
The behavior is weird. Swiping from Y to X seems to slow it down but swiping from X to Y always sets it to 8x. What's up with that?

**Review by AEFeinstein — CHANGES_REQUESTED** [review]
Please make the slider work in both directions

**Comment by dylwhich** [comment]
Huh, it is indeed very weird. That's especially strange because I totally ripped the logic out of paint, where it seems to work perfectly. I bet there's a bug there that doesn't manifest for whatever reason. I shall investigate and fix!

File: main/modes/mode_dance.c (hunk 1 above)
Comment (dylwhich) [reply main/modes/mode_dance.c:457]: Yeah, the screen gets enabled and updated during the entire swipe

**Comment by dylwhich** [comment]
I also removed 8x because it just feels _way_ too fast. 4x might even be a bit much still, particularly for the siren dance. But my swadge doesn't have jellybeans so maybe its not as actually as bad as it seems.

**Review by AEFeinstein — APPROVED** [review]

**Pull request merged**